src/codewhisperer/util/supplementalContext/editorFilesUtil.ts

```diff
@@ -37,7 +38,7 @@ export async function getRelevantFilesFromEditor(inputFileName: string, language
 }
 
 export function isRelevant(inputFileName: string, crossFileName: string, language: string) {
-    if (inputFileName === crossFileName) {
+    if (areEqual(undefined, inputFileName, crossFileName)) {
```

Comment:
nit: consider rename this `areEqual` function to something more meaningful and readable